src/cgame/cg_rocket_draw.cpp

```diff
@@ -3705,6 +3745,7 @@ void CG_Rocket_RegisterElements()
 	RegisterElement<BeaconOwnerElement>( "beacon_owner" );
 	RegisterElement<PredictedMineEfficiencyElement>( "predictedMineEfficiency" );
 	RegisterElement<BarbsHudElement>( "barbs" );
+	RegisterElement<CustomizableTextElement>( "text" );
```

Comment:
I still think we should call it `<quake>` or `<qtext>` and have the quake portion implied. Otherwise, it seems kinda pointless? If you want emoticons, you can just do `<img class=emoticon src="/emoticons/blah" />`.